### ISSUE TITLE
PersistentWindows tray icon disappears on hover, app crashes.

### ISSUE THREAD

**Issue by rocketfuelindustries** [kangyu-california/PersistentWindows#117]
PersistentWindows version 5.23.7804.24333
System is Windows 10 20H2 (19042.985) x64 with all latest updates.

NB: Recent change to the computer was installation of latest .NET Framework 5.0.2 Desktop 32-bit (to run a 32 bit app).

After starting PersistentWindows the app crashes if the mouse crosses the tray icon. It possibly crashes at other times (i.e. when I'm not at computer). This happens when started manually both in user context or as admin, and also when task is run in Task Scheduler (only tested admin).

Windows Application log shows exactly 50 Event Id 9999 within 3 seconds with info
 `PersistentWindows: Auto correct negative y screen coordinate`

This is followed by Event 1026 Source .NET Runtime
```
Application: PersistentWindows.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.BadImageFormatException
   at Ninjacrab.PersistentWindows.Common.Diagnostics.Log..cctor()

Exception Info: System.TypeInitializationException
   at Ninjacrab.PersistentWindows.Common.Diagnostics.Log.Trace(System.String, System.Object[])
   at Ninjacrab.PersistentWindows.Common.PersistentWindowProcessor.CaptureApplicationsOnCurrentDisplays(System.String, Boolean, Boolean)
   at Ninjacrab.PersistentWindows.Common.PersistentWindowProcessor.Start(Boolean)
   at Ninjacrab.PersistentWindows.SystrayShell.Program.Main(System.String[])
```

And followed by Event 1000 with
```
Faulting application name: PersistentWindows.exe, version: 5.23.7804.24333, time stamp: 0x609ede0b
Faulting module name: KERNELBASE.dll, version: 10.0.19041.964, time stamp: 0x812662a7
Exception code: 0xe0434352
Fault offset: 0x0000000000034b89
Faulting process ID: 0x3988
Faulting application start time: 0x01d756da2287b580
Faulting application path: C:\tools\PersistentWindows\PersistentWindows.exe
Faulting module path: C:\WINDOWS\System32\KERNELBASE.dll
Report ID: b712d0ae-fada-4b51-b740-5622c103b171
Faulting package full name: 
Faulting package-relative application ID: 
```

**Comment by kangyu-california** [comment]
Error "Auto correct negative y screen coordinate" has been removed since Release 5.6 around 2020/11. So that is not the cause.

I suspect the crash is caused by NLog.dll (a 3rd party utility) which might be incompatible with 32 bit environment.
The NLog.dll dependency has been removed in 5.24 (pre-release)
https://github.com/kangyu-california/PersistentWindows/releases/tag/5.24

I don't have 32 bit environment to verify, so this is the best I can give.
There is no guarantee that PW should work on 32 bit environment.

**Comment by rocketfuelindustries** [comment]
For clarification: This is a 64-bit environment, and was running 5.23.7804.24333 so "Auto correct negative y screen coordinate" might not have been entirely removed since Release 5.6, as I copied that error from yesterday's logs. The mention of 32 bit was because I had to install 32-bit .NET framework alongside 64-bit framework on this 64-bit Windows in order for an app (SDRSharp) to run.

Regardless, the problem has not occurred since installing 5.24. Thanks.
